.github/dependabot.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    # github-actions / path starts .github/workflows
+    directory: "/../../boilerplate"
```

Comment:
per the example https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#directory

it is relative from the root

so I guess should be

```yaml
- package-ecosystem: "github-actions"
  directory: "/boilerplate"
  schedule:
    interval: "daily"
```

Reply:
but I'm not sure if that will work https://github.com/dependabot/dependabot-core/blob/2f0db3e851ba2cc43d0b6dcd70da5e69d5b63eb6/github_actions/lib/dependabot/github_actions/file_fetcher.rb#L14

Reply:
i did similar in a personal repo and nothing changed :(

Reply:
Calling out the [same reference](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#directory):

> You define the directory relative to the root of the repository **_for all ecosystems except GitHub Actions. For GitHub Actions, set the directory to / to check for workflow files in .github/workflows._**

So:

```yaml
    directory: "/"
```

_should_ be sufficient.